src/server.cpp

```diff
@@ -2259,29 +2251,30 @@ void Server::SendBlocks(float dtime)
 	std::sort(queue.begin(), queue.end());
 
 	m_clients.lock();
-	for(u32 i=0; i<queue.size(); i++)
-	{
+	s32 max_blocks_to_send =
+			g_settings->getS32("max_simultaneous_block_sends_server_total");
+
+	for (const auto &block_to_send : queue) {
```

Comment:
Please use `PrioritySortedBlockTransfer` instead of `auto`, as we've already got a handy name for it.